Description: Don't install FindCtemplate.cmake.
Author: Felix Geyer <debfx-pkg@fobos.de>
Bug: http://sourceforge.net/tracker/?func=detail&aid=3002393&group_id=140662&atid=747132

diff -Nur kraft-0.40.orig/CMakeLists.txt kraft-0.40/CMakeLists.txt
--- kraft-0.40.orig/CMakeLists.txt	2010-04-13 18:09:10.000000000 +0200
+++ kraft-0.40/CMakeLists.txt	2010-05-16 14:23:17.065789688 +0200
@@ -12,7 +12,6 @@
 
 include_directories(${KDE4_INCLUDES} ${QT_INCLUDES} src)
 
-add_subdirectory(cmake)
 add_subdirectory(src)
 add_subdirectory(database)
 add_subdirectory(reports)
